src/ops/tag.rs

```diff
@@ -60,6 +60,8 @@ pub enum OpTag {
     BasicBlock,
     /// A control flow exit node.
     BasicBlockExit,
+    /// A lift node, which adds resource requirements to a wires type
+    Lift,
```

Comment:
Is this needed? Currently all leaf ops have `OpTag::Leaf`.

If the new tag is required:
- Change its parent to `OpTag::Lift => &[OpTag::Leaf],` (it's still a leaf).
- Return in in `LeafOp::tag`.

Reply:
Removed